test/functional/wallet_basic.py

```diff
@@ -54,7 +54,7 @@ def run_test(self):
         self.nodes[0].generate(1, self.signblockprivkey)
 
         walletinfo = self.nodes[0].getwalletinfo()
-        assert_equal(walletinfo['balance'], 50)
+        assert_equal(walletinfo['balance']['00'], 50)
```

Comment:
What is `00`?

Reply:
`00` means non-colored coin. This is defined in https://github.com/chaintope/tapyrus-core/blob/a4a42c2b06667377ab4b3b6aa7f872ad47d86d2d/src/coloridentifier.h#L40

Reply:
I see, but it is returned as the return value of json, a user-friendly value such as `TPC` or `Uncolored` is desirable.

Reply:
Yes, I agree with you.
I think `TPC` is better. Can I change the key to `TPC`?

Reply:
Yes please.

Reply:
Changed in 89fafd7